lib/game_box_web/live/upload_live.ex

```diff
@@ -103,20 +104,29 @@ defmodule GameBoxWeb.UploadLive do
     {:noreply, socket}
   end
 
-  defp get_game_upload_path(socket) do
-    disk_volume_path = Application.get_env(:game_box, :disk_volume_path)
+  defp get_paths(socket) do
+    get_path = fn result ->
+      case result do
+        [path] -> path
+        _ -> {:error, :no_files}
+      end
+    end
 
-    result =
-      consume_uploaded_entries(socket, :game, fn %{path: path}, _entry ->
-        dest = Path.join([disk_volume_path, Path.basename(path)])
-        File.cp!(path, dest)
-        {:ok, Path.basename(path)}
-      end)
+    art_result = handle_consume(socket, :artwork)
+    game_result = handle_consume(socket, :game)
 
-    case result do
-      [path] -> {:ok, path}
-      [] -> {:error, :no_files}
-    end
+    %{artwork: get_path.(art_result), game_path: get_path.(game_result)}
+  end
+
+  defp handle_consume(socket, atom) do
+    consume_uploaded_entries(socket, atom, fn %{path: path}, _entry ->
+      dest = Path.join([@disk_volume_path, Path.basename(path)])
```

Comment:
Interesting... are y'all using Fly.io's volumes for this? do they do some kind of magic replication?